data/core/units/elves/Outrider.cfg

```diff
@@ -17,9 +17,15 @@
+   
+Ryn was an elf of the northern Lintanir Forest who was particularly well-known for roaming the lands of Irdya and exploring uncharted territories. One of his many journeys took him eastward, where he rode into the rough foothills under the shadows of the dangerous Shakaar peaks. A little more ill-prepared than normal, he soon found himself beset by wild beasts and was forced to flee into the woods scattered across the mountainside. In one of the denser patches of forest, a pack of particularly persistent wolves killed his horse and nearly took his life, but he managed to escape them by climbing up an ancient tree where they could not follow. For three days and nights, they circled the roots of the enormous tree, watching with their crimson eyes, waiting for Ryn to tire. For three days and nights, he clung to life in the branches, fighting hunger and sleep as twilight and dawn painted the sky with the passage of time. 
 
-The speed of these soldiers allows the elves to strike when and where they choose when fighting on their home ground, a fact that has saved many of their number from death."
+On the fourth day, the sun did not rise. The wolves let out a mighty howl and suddenly, the trees around him paled to an ivory white, radiant with luminous silver from their slender branches to their numerous leaves. A great fog blanketed the forest, so thick that he could not see the ground from his vantage point and so dense that even as he fell from the branches, the mist gathered around him and suspended him in midair. Ryn heard a gentle voice call out to him in a tongue long forgotten and he looked up, catching a brief glimpse of butterfly wings and glowing eyes illuminating a feminine face that was composed more of faerie fire than physical flesh. She appeared only for a moment, then faded into the swirling mists, dispelling the vision with her and leaving him back on the ground, unharmed, but not alone. 
+
+Nearby, a white mare appeared between the trees, her coat as pure and alabaster as the trees from his vision had been. Ryn rose and approached cautiously; the only move she made was to bow to allow him to mount her more easily. The journey back to Lintanir Forest was an odd one — he could have sworn that it was at least a week's ride out into the wilderness, but the white mare took no more than two days to bear him back home. Upon returning, Ryn named her Isáwen and she remained with him for the rest of his days, carrying him faithfully on countless adventures over the passing years. When Ryn finally succumbed to old age over two centuries later, the white mare bore his body one last time back to the forest in the Shakaar mountains, where she disappeared and was never seen again.
+
+Although this legend is mostly told as a children's story, the fact remains that the white horses ridden by Elvish Outriders can only be found in the wild and are invariably female."
```

Comment:
Um ... wall of text ... and WTF? Do we really want a short story here? Maybe you want to do a campaign for the origin story for Outriders? Cut this down, a lot and get to the point quicker, OK?

Reply:
If someone else has the same opinion as this, then I will revert the commit and leave it at that. Alternatively, if someone else wants to write a new description for the Outrider, I can put it in this PR for convenience.

Reply:
I've pushed a revert of the Outrider at @nemaara's request.

The description _could_ stand with further revision though (it says that most humans would be "hard pressed" to equal Outrider's archery on solid ground, but even a lvl2 Longbowman has a better ranged attack than a lvl3 Outrider). Anyone has some text to suggest?

Reply:
Personally I didn't mind the short story, though perhaps a unit description isn't the best place for it. Maybe we can _allude_ to the story without including it in full?